pyproject.toml

```diff
@@ -13,55 +13,55 @@ repository = "https://github.com/canonical/pgbouncer-k8s-operator"
 
 [tool.poetry.dependencies]
 python = "^3.8.10"
-ops = "^2.0"
-cryptography = "^39.0.2"
-jsonschema = "^4.17.3"
-tenacity = "^8.2.2"
+ops = "2.1.1"
+cryptography = "39.0.2"
+jsonschema = "4.17.3"
+tenacity = "8.2.2"
```

Comment:
Renovate will update the lock and requirements but won't update `pyproject.toml` if the ^ or ~ requirement still holds true. I think this is easier to keep track of, since we wouldn't have to check the poetry lock or pip requirements to see the exact version in use.